app/views/companies/_company_members.html.haml

```diff
@@ -10,7 +10,7 @@
             %th= t('companies.show.members')
             %th= t('activerecord.models.business_category.other')
 
-        - applications.each do |application|
+        - applications.includes(:user, :business_categories, :shfapplications_business_categories).each do |application|
```

Comment:
The query should happen in the controller.

Reply:
I moved it to the controller.